omrmakefiles/rules.mk

```diff
@@ -207,7 +207,7 @@ endef
 endif
 
 DDR_SED_COMMAND := \
-    sed -n -e '/^DDRFILE_BEGIN /,/^DDRFILE_END /s/^/@/p'
+    sed -n -e '/^DDRFILE_BEGIN /,/^DDRFILE_END /s/^/@/' -e '/^@./p'
```

Comment:
Is there a similar change required for the CMake infrastructure?

Reply:
I don't believe DDR is supported with CMake yet.